lametro/templates/events/events.html

```diff
@@ -76,10 +76,13 @@ <h2><span>Upcoming {{ CITY_VOCAB.EVENTS }}</span>
                     </div>
                 {% endfor %}
 
-                <a href="" class="btn btn-salmon" id="more-upcoming-events"><i class="fa fa-fw fa-chevron-down"></i>Show all upcoming meetings</a>
-                <a href="" class="btn btn-salmon" id="fewer-upcoming-events"><i class="fa fa-fw fa-chevron-up"></i>Show fewer upcoming meetings</a>
+								{% if 'show' in request.GET and request.GET.show == 'future' %}
+									<a href="{% url 'events' %}" class="btn btn-salmon" id="fewer-upcoming-events"><i class="fa fa-fw fa-chevron-up"></i>Show fewer upcoming meetings</a>
+								{% else %}
+									<a href="{% url 'events' %}?show=future#upcoming-meetings" class="btn btn-salmon" id="more-upcoming-events"><i class="fa fa-fw fa-chevron-down"></i>Show all upcoming meetings</a>
+								{% endif %}
```

Comment:
What do you think about breaking this logic out into a partial that can be used for both future and past events?

Reply:
Hm, since the block is fairly small I'm thinking it might be clearer to keep it inline? I imagine the partial would also require an extra variable to know whether it's meant to render the upcoming or previous meeting buttons.

Reply:
Good point!